src/openapi/streaming/subscription.js

```diff
@@ -684,6 +695,7 @@ Subscription.prototype.STATE_SUBSCRIBED = 0x2;
 Subscription.prototype.STATE_UNSUBSCRIBE_REQUESTED = 0x4;
 Subscription.prototype.STATE_UNSUBSCRIBED = 0x8;
 Subscription.prototype.STATE_PATCH_REQUESTED = 0x10;
+Subscription.prototype.STATE_READY_FOR_UNSUBSCRIBE_BY_TAG = 0x20;

```

Comment:
i think this removed by mistake in this [PR](https://github.com/SaxoBank/openapi-clientlib-js/pull/99)

Reply:
Yeap, I came to the same conclusion during the TS migration.